aliases/available/vagrant.aliases.bash

```diff
@@ -2,13 +2,18 @@ cite 'about-alias'
 about-alias 'vagrant aliases'
 
 # Aliases
+alias vhl='vagrant hosts list'
+alias vscp='vaagrant scp'
+alias vsl='vagrant snapshot list'
+alias vst='vagrant snapshot take'
 alias vup="vagrant up"
 alias vupl="vagrant up 2>&1 | tee vagrant.log"
 alias vh="vagrant halt"
 alias vs="vagrant suspend"
 alias vr="vagrant resume"
 alias vrl="vagrant reload"
 alias vssh="vagrant ssh"
+alias vsshr="vagrant ssh $1 -- -t 'sudo su -; /bin/bash'"
```

Comment:
Yes, I had this as a function in my .bashrc, and thought that I could make it an alias. Again, should have tested this.  Thanks again for indulging me with all these errors.  I love this project, and am really happy to have the opportunity to contribute 🥇